torch/autograd/gradcheck.py

```diff
@@ -234,7 +269,9 @@ def gradcheck(
     check_sparse_nnz: bool = False,
     nondet_tol: float = 0.0,
     check_undefined_grad: bool = True,
-    check_grad_dtypes: bool = False
+    check_grad_dtypes: bool = False,
+    check_forward: bool = False,
```

Comment:
Everything I see "check_forward" I get confused:
- does it mean "check a forward pass?" or "forward-mode AD"?

Maybe we should clarify the name via `check_forward_mode_ad` or `check_jvp`